src/components/info-tile/examples/info-tile-progress.tsx

```diff
@@ -0,0 +1,64 @@
+import { Component, h, State } from '@stencil/core';
+import { InfoTileProgress } from '../info-tile.types';
+/**
+ * Displaying a progress bar
+ *
+ * By defining a numeric `progressValue`, you can display
+ * a circular progress bar to visualize more data on the component.
+ * This can for instance help illustrate how much of a
+ * set goal has been reached, which together with the `value` will help users
+ * get a better overview of the provided data.
+ *
+ * When the circular progress is shown, that would become the primary
+ * illustrative element on the component,
+ * which means the icon will be rendered smaller, only as a supportive
+ * contextual visual element.
+ *
+ * :::tip
+ * It is possible to customize the progress bar's suffix, but it is
+ * set to display the percentage sign (**%**) by default.
+ * :::
+ *
+ */
+@Component({
+    tag: 'limel-example-info-tile-progress',
+    shadow: true,
+    styleUrl: 'info-tile-progress.scss',
+})
+export class InfoTileProgressExample {
+    @State()
+    private progress: InfoTileProgress = {
+        value: 76,
+        maxValue: 100,
+        prefix: '↑',
+        suffix: 'sek',
+        displayPercentageColors: false,
+    };
```

Comment:
@LucyChyzhova seems you **HAVE to** add them, or you get an error. These should be optional.
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/35954987/203352383-016e429b-41b2-457f-8046-a182db68d08c.png">

Reply:
yes, added a fixup

Reply:
Fixed 👍